infra/gravity/cluster_status.go

```diff
@@ -2,45 +2,108 @@ package gravity
 
 import (
 	"context"
-	"time"
 
+	"github.com/cenkalti/backoff"
+
+	"github.com/gravitational/robotest/lib/constants"
 	sshutils "github.com/gravitational/robotest/lib/ssh"
 	"github.com/gravitational/robotest/lib/utils"
 	"github.com/gravitational/robotest/lib/wait"
-
 	"github.com/gravitational/trace"
 )
 
-// Status walks around all nodes and checks whether they all feel OK
-func (c *TestContext) Status(nodes []Gravity) error {
-	c.Logger().WithField("nodes", Nodes(nodes)).Info("Check status on nodes.")
-	ctx, cancel := context.WithTimeout(c.ctx, c.timeouts.Status)
-	defer cancel()
+// returns nil if the Gravity Status is the expected status or an error otherwise
```

Comment:
Nit: We follow Go Code Review Comments style guide for our code, according to which Godoc strings should start with the name of the thing they describe. I.e. `statusValidator declares a function type that returns nil if ....`

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Same for the Godoc strings below.

Reply:
Thanks.  I'll get these comments cleaned up.  I've also filed https://github.com/gravitational/robotest/issues/236 to help preempt this.